docs/courseadministration/addcontent/quiz_import.md

```diff
@@ -0,0 +1,63 @@
+![HPI Logo](../../img/HPI_Logo.png)
+
+# Quiz Import from Google Spreadsheets
+
+You can import the quiz structure from Google Spreadsheets.
+All you have to do is the following:
+
+### 1. Make a copy of this template
+We can provide you with a quiz template that you can copy
+:::image type="content" source="../../img/courseadministration/quizmanagement/quiz_template.png" alt-text="How our quiz template looks like":::
+Find the non-to-be-edited quiz structure template [here](https://docs.google.com/spreadsheets/d/1_b8wO9_HlrrHQxagYan1iM6BBK0XtnD6vESxmhnML5s/edit#gid=0)
```

Comment:
I would not add the edit link to a public source. If we share the spreadsheet publicly, we should create a new "read-only" link.

Reply:
ah ok, sure, I missed that!

Reply:
uhmm, it seems that, even if we are using a URL without `edit` in it, it gets resolved to a URL with `edit` ...even when if the document is _de facto_ view only -- That's confusing :/
![image](https://user-images.githubusercontent.com/12894481/119996102-9d68ee80-bfce-11eb-9356-2e36ff7f1c01.png)

Reply:
Okay, but the access level changed.